RockLib.Configuration.ObjectFactory.UnitTests/ConfigurationObjectFactoryTests.cs

```diff
@@ -469,6 +494,22 @@ public void CanBindToSimpleConstructorParameters()
         }
 
         [Fact]
+        public void CanBindToBindToByteArrayConstructorParameters()
```

Comment:
There should also be a test to verify that `byte[]` properties (not just constructor parameters) are supported.

Reply:
Right.  As far as all the other byte list types, I can.  Just never really use byte collections other than the array, so it's all i thought about.